sdks/wasm/src/driver.c

```diff
@@ -674,3 +680,142 @@ mono_wasm_enable_on_demand_gc (void)
 {
 	mono_wasm_enable_gc = 1;
 }
+
+MonoAssembly* get_zone_Info_assembly()
```

Comment:
Symbols should start `mono_` or be file-level `static`.
Some other prefixes are ok too, like `ves_icall_` or maybe this new `xamarin_` thing.
Functions that take no parameters should be `(void)`, not `()`, until/unless it is C++.
`()` in C means "accepts anything", `(void)` means "accepts nothing".
They mean the same thing in C++, and `(void)` means the same thing in both languages.

Reply:
fixed in recent commit